controller/api/public/stat_summary.go

```diff
@@ -303,7 +303,7 @@ func (s *grpcServer) trafficSplitResourceQuery(ctx context.Context, req *pb.Stat
 
 		for _, backend := range backends {
 			name := backend.Service
-			weight := backend.Weight.String()
+			weight := string(backend.Weight)
```

Comment:
I think this needs to be `weight := fmt.Sprintf("%d", backend.Weight)`, otherwise it will attempt to convert the Weight as an ascii value.

Reply:
Ahh thanks Alex, that fixed the tests!

Reply:
Ahh thanks Alex, that fixed the tests!